src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowCreate.cs

```diff
@@ -227,16 +172,125 @@ public override EditUpdateCellResult SetCell(int columnId, string newValue)
 
         #endregion
 
-        private string GetTableClause()
+        private ScriptBuildResult BuildInsertScript(bool forCommand)
```

Comment:
And what does `forCommand` mean?

Reply:
I thought I didn't need block comments for private methods?